Makefile

```diff
@@ -182,6 +182,9 @@ vendor: $(GO)
 	$(GO) mod tidy
 	$(GO) mod vendor
 
+auto-bumper:
+	$(GO) run $(shell ls tools/bumper/*.go |grep -v test) ${ARGS}
```

Comment:
consider please 2nd answer here
https://stackoverflow.com/questions/23695448/how-to-run-all-go-files-within-current-directory-through-the-command-line-mult